packages/cyberstorm/src/components/CommunityCard/CommunityCardSkeleton.module.css

```diff
@@ -0,0 +1,44 @@
+.root {
+    display: flex;
+    flex-direction: column;
+    gap: var(--space--12);
+  }
+  
+ /*  .card {

```

Comment:
The commented out part should be removed and the weird indentation in the rest of the file fixed.